knowrob_common/knowrob_common/src/main/java/org/knowrob/prolog/PrologInterface.java

```diff
@@ -62,17 +64,17 @@ public static void initJPLProlog(String initPackage) {
     	if(!initialized) {
 
     		try {
-    			Vector<String> args= new Vector<String>(Arrays.asList(jpl.fli.Prolog.get_default_init_args()));
+    			Vector<String> args= new Vector<String>(Arrays.asList(org.jpl7.fli.Prolog.get_default_init_args()));
```

Comment:
Could you import org.jpl7.fli.Prolog, and then use just Prolog.
I guess this could be good when a new JPL version is there
to avoid merge conflicts between branches